### ISSUE TITLE
Troubles reading five_plates plate run

### ISSUE THREAD

**Issue by deleenm** [jcbird/ppv#5]
I was going through the five_plates tutorial and I ran into an issue:

`ppv.fiveplates.available_plateruns()`
['2020.08.b.mwm-bhm',
 'first_drafts',
 '2020.09.x.bhm-mwm',
 'templates',
 '2020.10.z.bhm-mwm',
 '2020.09.zzztmp.bhm-mwm',
 '2020.08.a.bhm-mwm',
 '2020.09.y.bhm-mwm',
 '2020.10.x.bhm-mwm',
 'plate_data_archive.dat']

So the next line did not work:
`platerun_08x = ppv.fiveplates.Platerun('2020.08.x.mwm-bhm')`

Since that run no longer exists.
So I thought I would be clever, and run this:
`platerun_08x = ppv.fiveplates.Platerun('2020.08.b.mwm-bhm')`
and got the following error:
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
~/anaconda3/envs/ppv/lib/python3.8/site-packages/astropy/io/ascii/ui.py in _guess(table, read_kwargs, format, fast_reader)
    463             reader = get_reader(**read_kwargs)
--> 464             dat = reader.read(table)
    465             _read_trace.append({'kwargs': copy.deepcopy(read_kwargs),

~/anaconda3/envs/ppv/lib/python3.8/site-packages/astropy/io/ascii/basic.py in read(self, table)
    153         """
--> 154         out = super().read(table)
    155 

~/anaconda3/envs/ppv/lib/python3.8/site-packages/astropy/io/ascii/core.py in read(self, table)
   1173         # Get the table column definitions
-> 1174         self.header.get_cols(self.lines)
   1175 

~/anaconda3/envs/ppv/lib/python3.8/site-packages/astropy/io/ascii/core.py in get_cols(self, lines)
    587             else:  # No header line matching
--> 588                 raise ValueError('No header line found in table')
    589 

ValueError: No header line found in table

During handling of the above exception, another exception occurred:

InconsistentTableError                    Traceback (most recent call last)
<ipython-input-9-ed987b55adfa> in <module>
      1 #Define a platerun
----> 2 platerun_08x = ppv.fiveplates.Platerun('2020.08.b.mwm-bhm')

~/anaconda3/envs/ppv/lib/python3.8/site-packages/ppv-0.0.1-py3.8.egg/ppv/fiveplates.py in __init__(self, run_name)
    183             pass  # all is well, platerun available
    184         self.name = run_name
--> 185         self.platesummary = io.load_fiveplates_summary(run_name)
    186         self.fieldnames = self._get_fields()
    187 

~/anaconda3/envs/ppv/lib/python3.8/site-packages/ppv-0.0.1-py3.8.egg/ppv/data/io.py in load_fiveplates_summary(platerun)
     73     else:
     74         raise FileNotFoundError(os.fspath(summary_file))
---> 75     return ascii.read(os.fspath(summary_file), header_start=1,
     76                       format='commented_header')
     77 

~/anaconda3/envs/ppv/lib/python3.8/site-packages/astropy/io/ascii/ui.py in read(table, guess, **kwargs)
    285         # through below to the non-guess way so that any problems result in a
    286         # more useful traceback.
--> 287         dat = _guess(table, new_kwargs, format, fast_reader)
    288         if dat is None:
    289             guess = False

~/anaconda3/envs/ppv/lib/python3.8/site-packages/astropy/io/ascii/ui.py in _guess(table, read_kwargs, format, fast_reader)
    494                    '************************************************************************']
    495             lines.extend(msg)
--> 496             raise core.InconsistentTableError('\n'.join(lines))
    497 
    498 

InconsistentTableError: 
ERROR: Unable to guess table format with the guesses listed below:
Reader:FastCommentedHeader fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1
Reader:CommentedHeader delimiter: '|' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: '"'
Reader:CommentedHeader delimiter: '|' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: "'"
Reader:CommentedHeader delimiter: ',' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: '"'
Reader:CommentedHeader delimiter: ',' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: "'"
Reader:CommentedHeader delimiter: ' ' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: '"'
Reader:CommentedHeader delimiter: ' ' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: "'"
Reader:CommentedHeader delimiter: '\\s' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: '"'
Reader:CommentedHeader delimiter: '\\s' fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1 quotechar: "'"
Reader:CommentedHeader fast_reader: {'enable': True} fill_values: [('', '0')] header_start: 1

************************************************************************
** ERROR: Unable to guess table format with the guesses listed above. **
**                                                                    **
** To figure out why the table did not read, use guess=False and      **
** fast_reader=False, along with any appropriate arguments to read(). **
** In particular specify the format and any known attributes like the **
** delimiter.                                                         **
************************************************************************

**Comment by jcbird** [comment]
This sucks. the  structure of the five_plates repo JUST changed. See my issue here https://github.com/sdss/five_plates/issues/18#issue-715217135

I will update to the new structure after dinner. So sorry! This is the issue when trying to design a tool going after a moving target.

**Comment by jcbird** [comment]
Thanks @deleenm!  
Since 5975b3ef30156586c1ca332d7fb3f183903b0e80 , the tutorial should now work. Pull the latest commit and you'll be good to go methinks! Let me know.

**Comment by kevincovey** [comment]
I am very sorry -- I adjusted the file tree in advance of a new set of outputs that Felipe will push tomorrow, but did not realize I would be breaking this package.  

The first_drafts folder was an attempt to clean up the plateruns directory, by archiving a set of MWM sub-plateruns that are being merged back into a single platerun after triaging a number of fields that do not have enough BOSS standards to be designed.  This was intended to make the plateruns directory easier to navigate, and we didn't anticipate it causing problems as the platerun will have a new home tomorrow, but we did not remember that you are working to vet the plates in parallel with what we are doing.

In the future we will pick platerun names and stick with them, and they will live in the top level directory as they have. 

Sorry again for the frustration.

**Comment by eleonorazari** [comment]
I am going through the five_plates tutorial too, and I get a similar issue, although for me no plate runs seem to be available.
I get:
ppv.fiveplates.available_plateruns()
[]

My ppv_setup.ini looks like: 
[paths]
plate_dir = /home/zari/Documents/SDSS_Data/Plates
fiveplates_dir = /home/zari/Documents/SDSS_Data/Five_Plates

Is this right?

**Comment by jcbird** [comment]
@eleonorazari  this looks like you do not have the five_plates data on your disk. Check the README. You need to clone the five_plates repository manually  via git clone https://github.com/sdss/five_plates.git   .  You *may* need to be added to the SDSS github organization to access five_plates. Let me  know!

**Comment by jcbird** [comment]
@kevincovey thanks. In the light of day, I know we are all just trying to move quickly and do what's best. We will figure this out! I apologize for my frustration last night :)

**Comment by eleonorazari** [comment]
Hi @jcbird! Okay, I have tried to git clone  https://github.com/sdss/five_plates.git, and this is the output from my teminal:

git clone https://github.com/sdss/five_plates.git 
Cloning into 'five_plates'...
Username for 'https://github.com': eleonorazari
Password for 'https://eleonorazari@github.com': [well, I tried something... :D ]
remote: Repository not found.
fatal: repository 'https://github.com/sdss/five_plates.git/' not found

**Comment by jcbird** [comment]
Yep, you don't have access to the SDSS github repository yet. Let me raise the issue within five_plates and I'll see what the best solution for access is. I hope this gets resolved today and I will let you know as soon as possible!

**Comment by eleonorazari** [comment]
Thanks! :)

**Comment by jcbird** [comment]
EDIT: WAIT!!!! New plan! 

Ok, settled methinks :)

I have your github username from this comment. Joel wants me to send him a list of usernames to have access. So, this is on me for today! I will get access to you as soon as possible. Thanks! 

---- Original comment below this line ---- PLEASE IGNORE ---

**Comment by jcbird** [comment]
Going to close this since I think it's settled. Let me know if I closed it in error.
